easybuild/tools/filetools.py

```diff
@@ -1011,10 +1018,9 @@ def calc_block_checksum(path, algorithm):
     _log.debug("Using blocksize %s for calculating the checksum" % blocksize)
 
     try:
-        f = open(path, 'rb')
-        for block in iter(lambda: f.read(blocksize), b''):
-            algorithm.update(block)
-        f.close()
+        with open(path, 'rb') as fh:
```

Comment:
also use `_open_wrapper` here, for consistency?

Reply:
As discussed in Slack: Not using that function for binary files